ladderly-io/prisma/schema.prisma

```diff
@@ -52,27 +54,36 @@ model User {
   profileTopServices          String[]
   profileTopSkills            String[]
   profileYearsOfExperience    Int?
-  profileHighestDegree        String?
-  profilePicture              String   @default("")
-  residenceCountry            String   @default("")
-  residenceUSState            String   @default("")
-
-  accounts            Account[]
-  authoredVotables    Votable[]                       @relation("AuthoredVotables")
-  jobSearches         JobSearch[]
-  quizResults         QuizResult[]
-  role                RoleEnum                        @default(USER)
-  sessions            Session[]
-  subscriptions       Subscription[]
-  tokens              Token[]
-  transactions        Transaction[]
-  userChecklists      UserChecklist[]
-  uuid                String                          @default(uuid())
-  userChecklistItems  UserChecklistItem[]
-  votes               Vote[]
-  Lead                Lead[]
-  journalEntries      JournalEntry[]
-  practiceCompletions UserJournalPracticeCompletion[]
+
+  deletedAt DateTime?
+
+  // Add these relation fields:
+  accounts                       Account[]
+  sessions                       Session[]
+  tokens                         Token[]
+  userChecklists                 UserChecklist[]
+  userChecklistItems             UserChecklistItem[]
+  subscriptions                  Subscription[]
+  transactions                   Transaction[]
+  quizResults                    QuizResult[]
+  authoredVotables               Votable[]                       @relation("AuthoredVotables")
+  votes                          Vote[]
+  leads                          Lead[]
+  jobSearches                    JobSearch[]
+  journalEntries                 JournalEntry[]
+  userJournalPracticeCompletions UserJournalPracticeCompletion[]
+}
+
+enum ReminderFrequency {
+  NONE
```

Comment:
i guess we don't need both `NONE` here and also the boolean `journalReminderEnabled`